facebook/__init__.py

```diff
@@ -44,7 +44,9 @@
 __version__ = version.__version__
 
 FACEBOOK_GRAPH_URL = "https://graph.facebook.com/"
-FACEBOOK_OAUTH_DIALOG_URL = "https://www.facebook.com/dialog/oauth?"
+FACEBOOK_URL = "https://www.facebook.com/"
+FACEBOOK_OAUTH_DIALOG_URI = "dialog/oauth?"
+FACEBOOK_OAUTH_DIALOG_URL = FACEBOOK_URL + FACEBOOK_OAUTH_DIALOG_URI
```

Comment:
I try to keep it for backward capability. It not used anywhere, I will remove it